packages/common/src/types/certificate.ts

```diff
@@ -298,6 +301,23 @@ export interface ConfigUeCauseOfDeathList extends CertificateDataConfig {
   list: ConfigUeCauseOfDeathControl[]
 }
 
+export interface ConfigUeViewText extends CertificateDataConfig {
+  label?: string
+}
+
+export interface ConfigUeViewList extends CertificateDataConfig {
+  label?: string
+}
+
+export interface ConfigViewColumn {
+  id: string
+  title: string
```

Comment:
I suggest we try to follow the same structure that we have in our miro board:
`{`
`text: Sting`
`columns: [
            id: String
            title:  String
        ]`
`}`
This will allow us to create ViewTables with more than one column.

Reply:
If you look at ConfigUeViewTable you see that's exactly what we do! (but title as in Miro, instead of text, but maybe text is better

![image](https://user-images.githubusercontent.com/92843575/209674482-36409f30-7901-4cbe-96fd-e47275ad34f3.png)

Reply:
Awwww... :-) Nice!! 

I think it would make sense to use text instead of title, since thats what we usually call our "headers" 👍